code/modules/mob/living/silicon/silicon.dm

```diff
@@ -114,7 +114,7 @@
 
 	return 2
 
-/mob/living/silicon/apply_effect(var/effect = 0,var/effecttype = STUN, var/blocked = 0)
+/mob/living/silicon/apply_effect(var/effect = 0,var/effecttype = STUN, var/blocked = 0, var/negate_armor = 0)

```

Comment:
no implied vars, for petes sake.

Reply:
this is how it's defined in proc/apply_effect